main.go

```diff
@@ -33,10 +36,16 @@ func main() {
 	log.Info().Msg("String digitalstrom MQTT!")
 
 	ds := digitalstrom.New(config)
-	mqtt := digitalstrom_mqtt.New(&config.Mqtt, ds)
+	mqtt := digitalstrom_mqtt.New(&config.Mqtt, &config.Digitalstrom, ds)
 
 	ds.Start()
 	mqtt.Start()
 
-	time.Sleep(100 * 365 * 24 * time.Hour)
```

Comment:
Thanks :smile:  I know that I'm not the best when it comes to best practices with go.

Reply:
No problems. Is always important to have full control of the lifecycle of the program and gracefully handle the exit (closing connections, making notifications if needed,...)